src/darwin/Framework/CHIP/Matter.h

```diff
@@ -40,7 +40,7 @@
 #import <Matter/MTRDeviceController.h>
 #import <Matter/MTRDeviceControllerDelegate.h>
 #import <Matter/MTRDeviceControllerFactory.h>
-#import <Matter/MTRDeviceControllerStartupParameters.h>
```

Comment:
I think we need to keep including `MTRDeviceControllerStartupParameters.h` as well?

Reply:
That header no longer exists after this PR, so no.